src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/LincheckClassVisitor.kt

```diff
@@ -1222,14 +1221,17 @@ internal class LincheckClassVisitor(
             owner: String,
             name: String,
             itf: Boolean,
-            isAtomicFUMethod: Boolean
         ) {
+            val provideOwnerToBeforeMethodCall = opcode != INVOKESTATIC &&
```

Comment:
provideOwnerToBeforeMethodCall -> provideOwner + add a comment explaining the logic, please